docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.md

```diff
@@ -0,0 +1,336 @@
+---
+title: DigitalOcean App Platform Deployment - Strapi Developer Docs
+description: Learn in this guide how to deploy your Strapi application on DigitalOcean App Platform.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.html
+---
+
+# Deploy to the DigitalOcean App Platform
```

Comment:
> More a question than a suggestion: in most titles we used the "-ing" form for verbs, but I see here you chose not to do it. Does one sound better than the other for titles? 🤔

It is an accessibility issue: https://developers.google.com/style/headings

Reply:
Well, that's indeed a good reason, though it would have been better to share that with the team so we all discuss and decide together to change the way we write titles. If you're the only one to do that on your own, it just looks like it's not consistent. But if you opened a discussion with the team, it would allow everyone to learn and would make team work more efficient. Don't you think?